control-plane/subcommand/server-acl-init/servers.go

```diff
@@ -48,8 +50,8 @@ func (c *Command) bootstrapServers(serverAddresses []net.IPAddr, bootstrapToken,
 func (c *Command) bootstrapACLs(firstServerAddr, bootTokenSecretName string) (string, error) {
 	config := c.consulFlags.ConsulClientConfig().APIClientConfig
 	config.Address = firstServerAddr
-	consulClient, err := consul.NewClient(config,
-		c.consulFlags.APITimeout)
+	config.HttpClient = &http.Client{Timeout: 5 * time.Minute}
+	consulClient, err := consul.NewClient(config, c.consulFlags.APITimeout)
```

Comment:
Could you also keep the comment from before? I think it's useful.

Reply:
Yup!! Will do!